ui/pages/confirmation/templates/switch-ethereum-chain.js

```diff
@@ -83,7 +83,7 @@ function getValues(pendingApproval, t, actions) {
     onCancel: () =>
       actions.rejectPendingApproval(
         pendingApproval.id,
-        ethErrors.provider.userRejectedRequest(),
+        ethErrors.provider.userRejectedRequest().serialize(),
```

Comment:
This is called in `add-ethereum-chain.js` as well. So if we're still going the route of sending the error from the UI, it'll need to be serialized there as well. Presumably that confirmation suffers from the same problem.